dev-scripts/make-cli-executable.ts

```diff
@@ -0,0 +1,8 @@
+import { chmodSync } from 'fs';
```

Comment:
I'd call the directory just `scripts` instead of `dev-scripts` but that's just nitpicking :slightly_smiling_face: